### PR TITLE
Do template loading at template_include with priority 100 instead of 10

### LINKED ISSUE #2063: Divi Templates Ignore Membership Restrictions on Posts
### Reproduction Steps

+ HS-188870
+ Go to Divi --> Theme Builder and create a default website template and click the Save button
+ Create a global header, body and footer for the layout and save the layout
+ Create a new page and restrict access to that page to a membership
+ Navigate to the page while not logged in and you will see the page content is displayed
+ Go back to the Divi Builder and click the 3 dots above the default website template and click Disable Template
+ Navigate back to the page while not logged in and you will the access restricted notice is now displayed

### Expected Behavior

+ If a membership restriction is added to a page non-members should not be able to access the page

### Actual Behavior

+ If a Divi template is applied to the page the restriction is ignored if only "show message" is used
+ Has Alternative: The Redirect to Membership Page does work.

### Error Messages / Logs

+ Include any relevant error messages or log files
```
<!-- Paste error logs / backtraces below this line -->

```

### System and Environment Information

<details>
<summary>System Report</summary>

```
Wordpress
-------------------------------------------

Home Url: http://fresh.local
Site Url: http://fresh.local
Login Url: http://fresh.local/wp-login.php
Version: 5.9.2
Debug Mode: Yes
Debug Log: Yes
Debug Display: No
Locale: en_US
Multisite: No
Page For Posts: Not Set
Page On Front: Home (#2457) [http://fresh.local/]
Permalink Structure: /%postname%/
Show On Front: page
Wp Cron: Yes


Settings
-------------------------------------------

Version: 6.0.0
Db Version: 6.0.0
Course Catalog: Course Catalog (#4751) [http://fresh.local/courses/]
Membership Catalog: Membership Catalog (#4752) [http://fresh.local/memberships/]
Student Dashboard: Dashboard (#1491) [http://fresh.local/dashboard/]
Checkout Page: Purchase (#1490) [http://fresh.local/purchase/]
Course Catalog Per Page: 9
Course Catalog Sorting: menu_order,ASC
Membership Catalog Per Page: 9
Membership Catalog Sorting: menu_order
Site Membership: Not Set
Courses Endpoint: my-courses
Edit Endpoint: edit-account
Lost Password Endpoint: lost-password
Vouchers Endpoint: redeem-voucher
Autogenerate Username: yes
Password Strength Meter: yes
Minimum Password Strength: strong
Terms Required: yes
Terms Page: Affiliate Area (#3834) [http://fresh.local/affiliate-area/]
Checkout Names: required
Checkout Address: required
Checkout Phone: optional
Checkout Email Confirmation: yes
Open Registration: no
Registration Names: required
Registration Address: optional
Registration Phone: hidden
Registration Voucher: optional
Registration Email Confirmation: no
Account Names: required
Account Address: required
Account Phone: optional
Account Email Confirmation: yes
Confirmation Endpoint: confirm-payment
Force Ssl Checkout: no
Country: US
Currency: USD
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Decimals: 2
Trim Zero Decimals: no
Recurring Payments: yes
Email From Address: banana@example.com
Email From Name: fresh
Email Footer Text: 
Email Header Image: 
Cert Bg Width: 800
Cert Bg Height: 616
Cert Legacy Compat: no


Constants
-------------------------------------------

LLMS_REMOVE_ALL_DATA: undefined
LLMS_REST_DISABLE: undefined
LLMS_SITE_FEATURE_RECURRING_PAYMENTS: undefined
LLMS_SITE_IS_CLONE: undefined


Gateways
-------------------------------------------

Manual: Enabled
Manual Logging: no
Manual Order: 1


Server
-------------------------------------------

Mysql Version: 5.7.26
Php Curl: Yes
Php Default Timezone: UTC
Php Fsockopen: Yes
Php Max Input Vars: 1000
Php Max Upload Size: 8 MB
Php Memory Limit: 256M
Php Post Max Size: 8M
Php Soap: Yes
Php Suhosin: No
Php Time Limt: 30
Php Version: 7.4.2
Software: Apache
Wp Memory Limit: 40M


Browser
-------------------------------------------

HTTP USER AGENT: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.51 Safari/537.36


Theme
-------------------------------------------

Name: Divi
Version: 4.14.9
Themeuri: http://www.elegantthemes.com/gallery/divi/
Authoruri: http://www.elegantthemes.com
Template: 
Child Theme: No
Llms Support: No


Plugins
-------------------------------------------

LifterLMS: 6.0.0


Integrations
-------------------------------------------

BbPress: No
BuddyPress: No


Template Overrides
-------------------------------------------




```

</details>

This issue has be recreated:
+ [x] Locally
+ [x] On a staging site
+ [x] On a production website
+ [ ] With only LifterLMS and a default theme

### Browser, Device, and Operating System Information

+ Browser name and version
+ Operating System name and version
+ Device name and version (if applicable)

### PR DESCRIPTION
**Pull request by eri-trabiccolo**
## Description
Fixes #2063

## How has this been tested?
Tested with Divi  using reproduction steps in #2063 
I also tested the following themes ensuring no regression was introduced.

- [x] 2020
- [x] 2021
- [x] 2022
- [x] Astra
- [x] BuddyBoss
- [x] OceanWP
- [x] Kadence
- [x] GeneratePress

Any other theme I need to test?

## Types of changes
Compatibility fix (non-breaking change which fixes an issue) -->

## Checklist:
- [x] My code has been tested.
- [x] My code passes all existing automated tests. 
- [x] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

File: .changelogs/issue_2063-1.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+significance: patch
```
Comment (pondermatic) [inline .changelogs/issue_2063-1.yml:1]: A new filter is being added.
```suggestion
significance: minor
```

File: includes/class.llms.template.loader.php [hunk 2]
```diff
@@ -29,14 +29,26 @@ class LLMS_Template_Loader {
...
 		add_action( 'template_redirect', array( $this, 'hook_block_template_loader' ) );
 
+		/**
+		* Filters the template loading priority.
+		*
+		* Callback for the WP core filter `template_include`.
+		*
+		* @since [version]
+		*
+		* @param $priority The filter callback priority.
```
Comment (pondermatic) [inline includes/class.llms.template.loader.php:46]:
```suggestion
		* @param int $priority The filter callback priority.
```

**Review by pondermatic — APPROVED** [review]

**Review by thomasplevy — APPROVED** [review]

File: .changelogs/issue_2063-1.yml (hunk 1 above)
Comment (eri-trabiccolo) [reply .changelogs/issue_2063-1.yml:1]: I think you're right, thanks.

**Comment by eri-trabiccolo** [comment]
@thomasplevy I think @pondermatic review was correct.
will you add the suggestions by yourself?

**Comment by thomasplevy** [comment]
argh looks like he submitted those almost at the exact moment that I merged. No, I won't do it myself. You guys figure it out <3

**Pull request merged**